### PR TITLE
Fix Message erroring when content doesn't exist.

### LINKED ISSUE #83: keyerror 'content'
### Summary

Wrapper error 

### Reproduction Steps

Do literally anything and you'll get this error

### Minimal Reproducible Code

```python
import logging
import voltage
from voltage.ext import commands
from bot import evolve
logger = logging.getLogger(__name__)


class ping(commands.SubclassedCog):
    def __init__(self, bot: evolve):
        self.bot = bot
        self.name = "ping"


    @commands.command()
    async def ping(self, ctx):
        """Ping the bot"""
        embed = voltage.SendableEmbed(
            title="Ping",
            description="Hi I'm alive!",
        )
        await ctx.send(embed=embed)


def setup(bot: evolve) -> None:
    return ping(bot)
```

### Expected Results

The command responds just fine but this error is given in my console 

```py
Traceback (most recent call last):
  File "/home/lain/.local/lib/python3.10/site-packages/voltage/internals/ws.py", line 136, in handle_event
    await func(payload)
  File "/home/lain/.local/lib/python3.10/site-packages/voltage/internals/ws.py", line 161, in handle_message
    await self.dispatch("message", self.cache.add_message(payload))
  File "/home/lain/.local/lib/python3.10/site-packages/voltage/ext/commands/client.py", line 254, in dispatch
    await func(*args, **kwargs)
  File "/home/lain/.local/lib/python3.10/site-packages/voltage/ext/commands/client.py", line 291, in handle_commands
    return await self.commands[command].invoke(
  File "/home/lain/.local/lib/python3.10/site-packages/voltage/ext/commands/command.py", line 252, in invoke
    return await coro
  File "/mnt/c/Users/Lain/Desktop/Developer/revolt-evolve/cogs/ping.py", line 21, in ping
    await ctx.send(embed=embed)
  File "/home/lain/.local/lib/python3.10/site-packages/voltage/messageable.py", line 174, in send
    msg = self.cache.add_message(message)
  File "/home/lain/.local/lib/python3.10/site-packages/voltage/internals/cache.py", line 243, in add_message
    message = Message(data, self)
  File "/home/lain/.local/lib/python3.10/site-packages/voltage/message.py", line 130, in __init__
    self.content = data["content"]
KeyError: 'content'
```

### Actual Results

Given an error

### Checklist

- [X] I have searched the open issues for duplicates.
- [X] I have shown the entire traceback, if possible.
- [X] I have removed my token from display, if visible.

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by RGBCube**
## Summary

Closes #83 

## Checklist

- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [x] This PR fixes an issue.
- [ ] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, …)

### CODE REVIEW

**Comment by EnokiUN** [comment]
why are you using `""` instead of making it Null, also consider updating the base message payload type to use Optional[str]

File: voltage/message.py [hunk 1]
```diff
@@ -127,7 +127,7 @@ def __init__(self, data: MessagePayload, cache: CacheHandler):
         self.cache = cache
         self.id = data["_id"]
         self.created_at = ULID().decode(self.id)
-        self.content = data["content"]
+        self.content = data.get("content", "")
```
Comment (EnokiUN) [inline voltage/message.py:130]: Why are you using an empty string instead of making it Null?

**Review by EnokiUN — CHANGES_REQUESTED** [review]

File: voltage/message.py (hunk 1 above)
Comment (RGBCube) [reply voltage/message.py:130]: thats how dpy does it, and its better tbh

**Comment by toolifelesstocode** [comment]
This should also be changed.
https://github.com/EnokiUN/voltage/blob/main/README.rst#L34

As message content is no longer required, so it can not say it is obliged.

File: voltage/message.py (hunk 1 above)
Comment (ooliver1) [reply voltage/message.py:130]: dpy does it because discord always provides an empty string

Comment (EnokiUN) [reply voltage/message.py:130]: in addition to what olivier said, with voltage not using a None is just wrong

Comment (RGBCube) [reply voltage/message.py:130]: oh ok, ill change it when i get home

**Pull request closed without merging**